README.md

```diff
@@ -62,6 +62,8 @@ build directory. This will create a CD image with a filename `bootcd.iso`.
 
 See ["Building ReactOS"](http://www.reactos.org/wiki/Building_ReactOS) for more details.
 
+You can always download fresh binary builds of bootable images from the ["special page"](https://www.reactos.org/getbuilds/)
+
```

Comment:
I'd argue `Daily Builds` (or anything similar) is much better than "special page".

Reply:
fixed